ci/ci/utils.py

```diff
@@ -1,9 +1,38 @@
+import datetime
 import secrets
 import string
+from typing import Dict, List, Optional
+
+from gear import Database
 
 
 def generate_token(size=12):
     assert size > 0
     alpha = string.ascii_lowercase
     alnum = string.ascii_lowercase + string.digits
     return secrets.choice(alpha) + ''.join([secrets.choice(alnum) for _ in range(size - 1)])
+
+
+async def add_deployed_services(
+    db: Database,
+    services_per_namespace: Dict[str, List[str]],
+    expiration_time: Optional[datetime.datetime],
+):
+    expiration = expiration_time.strftime('%Y-%m-%d %H:%M:%S') if expiration_time else None
```

Comment:
Is there a reason why you decided to use timestamps rather than BIGINT? Can you double check that the time zones here are going to be consistent such as with daylight savings?

Reply:
I think this was an oversight on my point in terms of consistency with the rest of our database infrastructure. I went with timestamp because it seemed easiest to "add a day" to. Happy to use BIGINT and `time_msecs` if that would be more consistent.

Reply:
I think either is fine as long as we're okay if it's not exactly 24 hours when there's daylight savings time. We do use DATETIME with session ids I believe.